### ISSUE TITLE
Suggested kits

### ISSUE THREAD

**Issue by larsbrinkhoff** [saildart/waits-disk-kits#1]
In addition to the 1974 based disk images, I'd like to suggest some additional kits.  Some of them may be a pipe dream, but it's always fun to make a wish list.

- The earliest workable set of files from 1972.
- The last KA10 system from May 1976, using the BBN pager for larger physical memory.
- Something from the early KL10 era: 1976-1983.  Maybe 1979 with the Samson box and some CCMRA software.  Or maybe 1982/83 with the sailboat WHOPHN.
- Something from the KL10+RH20 era: 1983-1990.

**Comment by larsbrinkhoff** [comment]
FYI @rcornwell, see above.

**Comment by bgbaumgart** [comment]
Yes @larsbrinkhoff once I have the WAITS 6.17 beachhead secured ( spanning 1974-06-26 to 1974-11-29 ) it will be easier to build further disk images for earlier and later points in time. I now have the source [J17,SYS] compiling (assembling) and loading correctly - but I had to borrow the COMCSS and the OUTER from [K17,SYS] and set the VODcoder voice device count to zero. There are numerous not so subtle differences in the software over the two decades - so that a Digital Curator ( the Time Lord ) has to find a compatible selection of files to place in each disk kit.

**Comment by larsbrinkhoff** [comment]
Thank you @bgbaumgart, that sounds great!  I mainly wanted to start a discussion.  Maybe you already have a clear picture of some points in time that are highlights in SAIL history.  Certainly I expect there is some patchwork necessary to make a working kits.  I'm ready to put my money where my mouth is by doing some of that work.  (I tried to make a consistent 8.01 monitor source kit, but the 1974 tool set wasn't able to assemble it.)
